### PR TITLE
enable config driven multi az

### PR DESCRIPTION
**Pull request by ptolemy428**
## Approvals

- [x] Verified
- [x] @TimMoore
- [ ] @hebbo
- [x] @ptolemy428
- [x] @iiman

### CODE REVIEW

**Comment by ptolemy428** [comment]
<a href="https:&#x2F;&#x2F;trello.com&#x2F;c&#x2F;PPNWyQr7&#x2F;443-create-cf-stacks-for-deployment-of-waffle-importer"> Create CF stacks for deployment of waffle Importer</a>

**Comment by iiman** [comment]
👍

File: cloudformationcore/load_balancer.py [hunk 1]
```diff
@@ -18,7 +18,7 @@ def create_internal_load_balancer(custom_resource, subnet_keys,
                 Protocol="HTTP", ),
         ],
         HealthCheck=elb.HealthCheck(
-            Target=Join("", ["HTTP:", "80", "/"]),
+            Target=Join("", ["HTTP:", "80", "/status"]),
```
Comment (TimMoore) [inline cloudformationcore/load_balancer.py]: Will we enforce this convention for all services?

**Comment by TimMoore** [comment]
Looks good 🚢

File: cloudformationcore/load_balancer.py (hunk 1 above)
Comment (ptolemy428) [inline cloudformationcore/load_balancer.py]: Yes, we should. All micro services (from data sourcery team) use this convention. Had a conversation with influx team previously on this as well. Team can still vote to change this in the future if they can justify their reasons.

**Pull request merged**